mmv1/products/datastream/Stream.yaml

```diff
@@ -1149,6 +1374,94 @@ properties:
+              description: |
+                SQL Server database.
+              properties:
+                - !ruby/object:Api::Type::String
+                  name: 'schema'
+                  required: true
+                  description: |
+                    Schema name.
+                - !ruby/object:Api::Type::Array
+                  name: 'tables'
+                  min_size: 1
+                  description: |
+                    Tables in the database.
+                  item_type: !ruby/object:Api::Type::NestedObject
+                    description: |
+                      SQL Server table.
+                    properties:
+                      - !ruby/object:Api::Type::String
+                        name: 'table'
+                        required: true
+                        description: |
+                          Table name.
+                      - !ruby/object:Api::Type::Array
+                        name: 'columns'
+                        min_size: 1
+                        description: |
+                          SQL Server columns in the schema. When unspecified as part of include/exclude objects, includes/excludes everything.
+                        item_type: !ruby/object:Api::Type::NestedObject
+                          description: |
+                            SQL Server Column.
+                          properties:
+                            - !ruby/object:Api::Type::String
+                              name: 'column'
+                              description: |
+                                Column name.
+                            - !ruby/object:Api::Type::String
+                              name: 'dataType'
+                              description: |
+                                The SQL Server data type. Full data types list can be found here:
+                                https://learn.microsoft.com/en-us/sql/t-sql/data-types/data-types-transact-sql?view=sql-server-ver16
+                            - !ruby/object:Api::Type::Integer
+                              name: 'length'
+                              output: true
+                              description: |
+                                Column length.
+                            - !ruby/object:Api::Type::Integer
+                              name: 'precision'
+                              output: true
+                              description: |
+                                Column precision.
+                            - !ruby/object:Api::Type::Integer
+                              name: 'scale'
+                              output: true
+                              description: |
+                                Column scale.
+                            - !ruby/object:Api::Type::String
+                              name: 'encoding'
+                              output: true
+                              description: |
+                                Column encoding.
```

Comment:
```suggestion
```

Reply:
Removed in https://github.com/GoogleCloudPlatform/magic-modules/commit/939b4f1e8962c3ac939f24d2fc98170f48bdc4d7